node/cmd/logscribe/main.go

```diff
@@ -20,8 +21,13 @@ func main() {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	err := logscribe.Run(ctx)
+	logscribe, err := logscribe.New(ctx, logscribe.WithProcessLogsInterval(2*time.Second))
```

Comment:
is there a specific reason you've set into 2 seconds process interval?

Reply:
oh no! that shouldn't have been there, it was for local testing, will remove it

Reply:
I think we can take it longer such as 30 seconds

Reply:
isn't processlogsinterval an interval to store logs in to db?

Reply:
awh I misunderstood, so processLogsInterval is interval for processing and removing logs

let's rename it into more specific one such as readAndRemoveInterval

Reply:
we can set bulkStoreInterval into something such as 30 seconds or 1 min, I remember you said it was 3 sec?